docs/api-reference/carto/quadbin-heatmap-tile-layer.md

```diff
@@ -0,0 +1,104 @@
+# QuadbinHeatmapTileLayer (Experimental)
+
+`QuadbinHeatmapTileLayer` is a layer for visualizing tiled data described using the [Quadbin Spatial Index](https://docs.carto.com/data-and-analysis/analytics-toolbox-for-bigquery/key-concepts/spatial-indexes#quadbin) using a heatmap. 
```

Comment:
Minor wording suggestion, to avoid repetition of "using".

```suggestion
`QuadbinHeatmapTileLayer` is a layer for visualizing tiled data indexed with a [Quadbin Spatial Index](https://docs.carto.com/data-and-analysis/analytics-toolbox-for-bigquery/key-concepts/spatial-indexes#quadbin) using a heatmap. 
```